ui/resetpassword.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE HTML>
```

Comment:
This currently opens as a page instead of a modal, which leads to the form looking quite strange as the `well` element stretches all the way across the page.